largestPrimeFactor.js

```diff
@@ -1,6 +1,22 @@
-exports.largestPrimeFactor = function(n){
+exports.largestPrimeFactor = function (n) {
   var primeNumber = 0;
   // do your work here
+  let primeFactors = [];
 
+  while(n%2 === 0){
+    primeFactors.push(2);
+    n = n/2;
+  }
+
+  let squareRootNum = Math.sqrt(n);
+  for(let i=3; i<=squareRootNum; i++){
+    if(n%i === 0){
```

Comment:
I think this is missing a while loop here. You need a while loop to keep pulling any of the same prime factors out of the number. You can take a look at this for reference: https://js-algorithms.tutorialhorizon.com/2015/09/27/find-all-the-prime-factors-for-the-given-number/